### DISCUSSION TITLE
Preferences/Settings files.exclude symlinks

### DISCUSSION THREAD

**Discussion by digitalnumeric** [eclipse-theia/theia#14009] [Q&A] [answered]
I can see the symlink decorator icons as per this: [Symlink Decorator Icon](https://github.com/eclipse-theia/theia/pull/10439)
I would like to hide symlinks similar to this: [Exclude Symlinks from Display](https://github.com/microsoft/vscode/issues/74335)
`{
    "files.exclude": {
        "**/.git": true,
        "symlinks": true
    }    
}`
Is the above the correct way to hide symlinks in theia?
Thank you.

**Comment by msujew** [comment, marked as answer]
Hey @digitalnumeric,

as far as I can tell, https://github.com/microsoft/vscode/issues/74335 was never actually implemented, right? It was marked as out-of-scope.

I don't think that Theia currently supports a setting to exclude symlinks in the explorer. You can create a feature proposal for that if you want to. Note that the proposed solution with `{ "files.exclude": { "**/.git": true, "symlinks": true } }` doesn't seem like a good idea, since it's pretty ambiguous.

**Reply by digitalnumeric** [reply]
I see now that you are correct and it wasn't implemented.  Thank  you for the reply.
